artifacts/definitions/Linux/Events/ProcessStatuses.yaml

```diff
@@ -0,0 +1,40 @@
+name: Linux.Events.ProcessStatuses
+description: |
+  This artifact collects new processes created by non-root users from
+  the Linux kernel audit events.
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT_EVENT
+
+required_permissions:
+  - EXECVE
+
+sources:
+  - query: |
+
+     LET proc_stat_rules = ("-a always,exit -F arch=b64 -S execve -F euid!=0 -F auid!=-1 -k vrr_proexec", "-a always,exit -F arch=b32 -S execve -F euid!=0 -F auid!=-1 -k vrr_proexec")
```

Comment:
This rule seem to log every single command executed by non-root users, even if the command with exist code other than zero (0). Moreover, in audit.log, it's showing the command as `success`  or `exit=0`, which does not seem to reflect the actual exit code of the command itself.